keepassxc-browser/content/form.js

```diff
@@ -158,6 +158,15 @@ kpxcForm.onSubmit = async function(e) {
     } else if (kpxc.credentials.length === 1) {
         // Single entry found for the page, use the username of it instead of an empty one
         usernameValue = kpxc.credentials[0].login;
+    } else {
```

Comment:
I just realized that maybe I should have used the following:

```suggestion
    } else if (kpxc.credentials.length > 1) {
```

Is there a chance that we get here when there's zero credentials?

Reply:
In theory yes, if a login has been made in the previous page and the current page has a different URL that is not matched with the selected entry at all. A rare case though. Here the `username` value is of course only used for the banner.

If it's a first time user enters credentials to a page, there are no credentials (yet). This `else` should cover that also.